src/main/docker/Dockerfile

```diff
@@ -23,23 +23,14 @@ ENV LANG=en_US.utf8
 # Update the OS packages, install cURL, postgreSQL client and dejavu-fonts
 RUN zypper -n refresh && \
     zypper -n update && \
-    zypper -n install curl postgresql dejavu-fonts && \
+    zypper -n install curl postgresql dejavu-fonts sudo && \
     zypper -n clean --all
 
-# Install gosu
-RUN gpg --batch --keyserver-options http-proxy=${env.HTTP_PROXY} --keyserver hkps://keys.openpgp.org \
-      --recv-keys B42F6819007F00F88E364FD4036A9C25BF357DD4 && \
-    curl -o /usr/local/bin/gosu -SL "https://github.com/tianon/gosu/releases/download/1.12/gosu-amd64" && \
-    curl -o /usr/local/bin/gosu.asc -SL "https://github.com/tianon/gosu/releases/download/1.12/gosu-amd64.asc" && \
-    gpg --batch --verify /usr/local/bin/gosu.asc /usr/local/bin/gosu && \
-    rm /usr/local/bin/gosu.asc && \
-    chmod +x /usr/local/bin/gosu
-
 # Add scripts to be executed during startup
 COPY startup /startup
-ADD https://raw.githubusercontent.com/CAFapi/caf-common/v1.19.0/container-cert-script/install-ca-cert.sh \
-    /startup/startup.d/install-ca-cert-base.sh
-RUN chmod +x /startup/* /startup/startup.d/*
+COPY cert-scripts/install_ca_cert.sh /startup/startup.d/install_ca_cert-base.sh
+RUN chmod -R +rx /startup/* /startup/startup.d/*
+COPY permissions/caf /etc/sudoers.d/caf
```

Comment:
I'm don't really like the file being called `caf`.  It doesn't project that the permissions are required for the `install-ca-cert-base.sh` script.  Can we change the filename to `install-ca-cert-base`?  And can we change the folder name from `permissions` to `sudoers.d`?